sql/sys_vars.cc

```diff
@@ -1607,6 +1607,13 @@ static Sys_var_bool Sys_binlog_rows_query(
     DEFAULT(false), NO_MUTEX_GUARD, NOT_IN_BINLOG,
     ON_CHECK(check_session_admin));
 
+static Sys_var_bool Sys_binlog_ddl_query(
+    "binlog_ddl_query_log_events",
```

Comment:
this option is not about logging DDL events to binlog, we already do that.
Can we use the name that the option does?
binlog_ddl_retain_comments?
binlog_ddl_disallow_query_rewrite?

Any others that u or @venkatesh-prasad-v can think of?

Reply:
As discussed over slack, I prefer naming it to `binlog_ddl_skip_rewrite`.

Reply:
Ok with it :+1: